app/models/approval_status.rb

```diff
@@ -77,6 +83,14 @@ def evaluation_threshold?
       true
     end
 
+    def non_voting_members_present?
+      non_voting_member = false
+      committee_members.each do |member|
+        non_voting_member = true unless member_voted?(member)
+      end
+      non_voting_member
+    end
```

Comment:
One other possibility for writing this:
`def non_voting_members_present
    committee_members.any? {|m| !member_voted?(m) }
end`